### DISCUSSION TITLE
Possibility to upload to cloud storage (with autoremove)

### DISCUSSION THREAD

**Discussion by nelutzu777** [alienatedsec/yi-hack-v5#227] [Ideas]
I'm not an expert to give my opinion, but I would really like the possibility to upload on MEGA.NZ, GOOGLE PHOTOS or other free storage. Since the application does not allow download of the file, but rather the capture of the film, it makes this option absolutely necessary.

**Comment by daich** [comment]
there is the ability to upload to an FTP server so if your cloud storage supports FTP that would work
